upup/pkg/fi/cloudup/bootstrapchannelbuilder_test.go

```diff
@@ -53,20 +53,6 @@ func TestBootstrapChannelBuilder_BuildTasks(t *testing.T) {
 	runChannelBuilderTest(t, "awsiamauthenticator", []string{"authentication.aws-k8s-1.12"})
 }
 
-func TestBootstrapChannelBuilder_PublicJWKS(t *testing.T) {
```

Comment:
Is there are reason to drop this test?

Reply:
Yeah. This PR only supports using s3 as store for jwks, so this fails because filesystem VFS cannot be used for https.